resources/views/admin/blocks/image.blade.php

```diff
@@ -1,6 +1,23 @@
 @php
     $currentUrl = explode('/', request()->url());
     $type = $currentUrl[5] ?? null;
+    $options = [];
+    $options[] = [
+        'value' => 's',
+        'label' => 'Small'
+    ];
+
+    if ($type !== 'digitalPublications') {  // replace $condition with your actual condition
```

Comment:
What does this comment mean?

Reply:
I probably left this one in lmao

Reply:
This one was me! Glad for the PR review.